modules/navier_stokes/src/materials/MixingLengthTurbulentViscosityMaterial.C

```diff
@@ -0,0 +1,70 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MixingLengthTurbulentViscosityMaterial.h"
+
+registerMooseObject("NavierStokesApp", MixingLengthTurbulentViscosityMaterial);
+
+InputParameters
+MixingLengthTurbulentViscosityMaterial::validParams()
+{
+  InputParameters params = ADMaterial::validParams();
+  params.addClassDescription("Computes the material corresponding to the turbulent viscosity"
+                             "for the mixing length model.");
```

Comment:
change description
this computes the total viscosity not just the turbulent viscosity